internal/db/schema/migrations/postgres/14/07_wh_credential_dimension.up.sql

```diff
@@ -0,0 +1,49 @@
+begin;
+  alter table wh_session_accumulating_fact
+    add column credential_group_key wh_public_id;
```

Comment:
Yea, I wasn't sure how we wanted to handle this. If I am following this correctly, there should always be a credential group, since it does not make sense to me that you would get a session without any credentials, but I wasn't sure if that was enforced elsewhere and would be safe to enforce here as well. It also means we need to update existing `wh_session_accumulating_facts`, but I am not sure what we would set the value to.

Reply:
Ah, I see you cover this in your overall review comment.